sim/deploy/run.py

```diff
@@ -62,11 +63,18 @@ class MujocoWorld(World):
     def __init__(self, cfg: RobotConfig):
         self.cfg = cfg
         self.model = mujoco.MjModel.from_xml_path(str(self.cfg.robot_model_path))
+
         self.model.opt.timestep = self.cfg.dt
+
         # First step
         self.data = mujoco.MjData(self.model)
+        self.data.qpos = self.model.keyframe("default").qpos
+
         mujoco.mj_step(self.model, self.data)
 
+        # Set self.data initial state to the default standing position
+        # self.data.qpos = StompyFixed.default_standing()
```

Comment:
Put TODO flag - the keyframe and default_standing should produce the same results soon